### DISCUSSION TITLE
Do you know when flat route will be officially in the core of Remix?

### DISCUSSION THREAD

**Discussion by juniorforlife** [kiliman/remix-flat-routes#45] [Q&A]
I read it from the [doc](https://remix.run/docs/en/1.14.3/file-conventions/route-files-v2) that 

> You can opt-in to the new route file naming convention with a future flag in Remix config. It will be the default behavior in the future when v2 ship 

Do you happen to know when v2 will be shipped?

**Comment by kiliman** [comment]
Hi. I'm not sure when v2 will be shipped, but pretty much all the v2 features can be enabled using future flags.

NOTE: v2 routing differs slightly from this package. `remix-flat-routes` added some enhancements that make it easier to use. So even if you migrate to v2, you can continue to use this package.

**Reply by HananoshikaYomaru** [reply]
I am sorry I didn't read the full doc. What is the difference between remix flat routes and the remix core v2? In term of route convention or functional capabilities
